linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/ubuntu2004.yml

```diff
@@ -0,0 +1,29 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Uncomment banner-message-enable for Login Warning Banner"
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)